database/variables.tf

```diff
@@ -33,7 +33,14 @@ variable "tags" {
 
 variable "json_params" {
   description = "A JSON string of arbitrary parameters"
```

Comment:
I missed that the actual parameter is expected to be the map, rather than the string! In that case we should definitely update the description here.